test/input_gen/genModelsRecurrent_v2.py

```diff
@@ -455,4 +483,13 @@ def forward(self, inputs, labels):
         name="grucell_stacked",
     )
 
+    unroll_for, num_grucell, batch_size, unit, feature_size, iteration, = [2, 1, 3, 2, 2, 2]
+    record_v2(
+        GRUCellFC(unroll_for=unroll_for, num_grucell=num_grucell),
+        iteration=iteration,
+        input_dims=[(batch_size, feature_size)] + [(batch_size, unit) for _ in range(num_grucell)],
+        label_dims=[(batch_size, unroll_for, unit)],
+        name="grucell_fc",
+    )
+
```

Comment:
```suggestion
        input_dims = [(batch_size, feature_size)] + [(batch_size, unit) for _ in range(num_grucell)],
        label_dims = [(batch_size, unroll_for, unit)],
        name = "grucell_fc",
    )
```

Reply:
Changed. I think python script formatter might be needed. Could you recommend one if you have in your mind?